index.html

```diff
@@ -940,28 +777,61 @@ <h2 class="font-weight-bold text-black">Contact Us</h2>
         </div>
       </div>
     </div> -->
-      <!-- ieee-ctf? /images/n00b.wav-->
-      <script src="js/jquery-3.3.1.min.js"></script>
-      <script src="js/jquery-ui.js"></script>
-      <script src="js/popper.min.js"></script>
-      <script src="js/bootstrap.min.js"></script>
-      <script src="js/owl.carousel.min.js"></script>
-      <script src="js/jquery.countdown.min.js"></script>
-      <script src="js/jquery.magnific-popup.min.js"></script>
-      <script src="js/bootstrap-datepicker.min.js"></script>
-      <script src="js/aos.js"></script>
-      <script src="js/jquery.sticky.js"></script>
-      <script src="js/jquery.easing.1.3.js"></script>
-
-      <script src="js/jquery.fancybox.min.js"></script>
-      <script src="js/main.js"></script>
+    <!-- ieee-ctf? /images/n00b.wav-->
+
+    <button class="chatbot-button"><img src="images/chat.svg"></button>
+
+    <div class="chatbot-wrap" style="display: none;">
+      <div class="chatbot-nav text-center"><img class="chatbot-nav-ieee-logo" src="images/main_logo_ieee.png"> <img class="chatbot-nav-cross" src="images/cross.svg">
```

Comment:
Can we use a higher quality logo asset? it is getting a bit blurry for me
![image](https://user-images.githubusercontent.com/54478821/128075596-a69e2e28-e8ce-4b86-bdc3-623ab1a86d73.png)